### PR TITLE
feat: 1193 adds fund grantees tab and contract interaction for payout(s)

### PR DESCRIPTION
**Pull request by KurtMerbeth**
This addresses #1193 and #1151

### CODE REVIEW

**Comment by codenamejason** [comment]
There is one missing piece to this being completed. We have a `View` link to view the payout transaction but I am not sure where to get the hash to append to the link uri.

File: packages/round-manager/src/features/round/ViewFundGrantees.tsx [hunk 1]
```diff
@@ -82,31 +89,45 @@ const TabApplicationCounter = tw.div`
...
+  // );
 
   const mapMatchingStatsDataToGranteeFundInfo = (matchingStatsData: MatchingStatsData[]): GranteeFundInfo[] => {
     return matchingStatsData.map((matchingStatData) => ({
       project: matchingStatData.projectName ?? "",
       walletAddress: matchingStatData.projectPayoutAddress,
       matchingPercent: (matchingStatData.matchPoolPercentage * 100).toString(),
       payoutAmount: matchingStatData.matchAmountInToken,
+      status: "Success",
+      hash: "0x1234567890", // todo: where to get this from?
```
Comment (bhargavaparoksham) [inline packages/round-manager/src/features/round/ViewFundGrantees.tsx:118]: @codenamejason txHash data for payouts is available in the subgraph

File: packages/round-manager/src/features/round/ViewFundGrantees.tsx [hunk 2]
```diff
@@ -82,31 +89,45 @@ const TabApplicationCounter = tw.div`
...
     }));
   };
 
   useEffect(() => {
-
+    setPrice(1808.04);
```
Comment (bhargavaparoksham) [inline packages/round-manager/src/features/round/ViewFundGrantees.tsx:123]: This needs to be changed back.

**Review by bhargavaparoksham — DISMISSED** [review]

File: packages/round-manager/src/features/api/payoutStrategy/merklePayoutStrategy.ts [hunk 3]
```diff
@@ -161,26 +161,45 @@ export const useGroupProjectsByPaymentStatus = (
...
       };
 
-      const paidProjectIds = (await paidProjectsFromGraph).map(
-        (project) => project.id
+      const paidProjects: Payout[] = await paidProjectsFromGraph;
+      const payoutAddress = paidProjects.map((project) =>
+        project.grantee.toLocaleLowerCase(),
       );
 
+      console.log("===> paidProjectsFromGraph", paidProjectsFromGraph);
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/api/payoutStrategy/merklePayoutStrategy.ts:180]:
```suggestion
```

**Review by thelostone-mc — APPROVED** [review]

**Comment by hmrtn** [comment]
What is your testing methodology here? Can you provide a screenshot of the table? I cannot get the table to display when running locally.

**Comment by codenamejason** [comment]
Screenshot 2023-04-01 at 10 13 12 AM
Screenshot 2023-04-01 at 10 13 21 AM

**Comment by thelostone-mc** [comment]
@hmrtn To test this out. 
You'd need a round which is completed with approved applications and stuff. 
This works only for round 0x5f437a92741be0f8bd6342668f1feb4414a9d5cc as we would need the API service wrapped up to wire in the calculations (you'll see a stub variable in the code)

**Pull request merged**